gemrb/core/GUI/GameControl.cpp

```diff
@@ -2079,6 +2075,15 @@ void GameControl::InitFormation(const Point &clickPoint)
 	SetCursor(core->Cursors[IE_CURSOR_USE]);
 }
 
+bool GameControl::TryDefaultTalk(Point p) const
+{
+	Actor *targetActor = core->GetGame()->GetCurrentArea()->GetActor(p,
```

Comment:
Should be const. But also, don't we have lastActor that holds what the mouse is over? Or is that just in other GameControl functions?

And if it's thus unavoidable, please split this into three lines, fetch the game, the map, then the actor, so the call itself won't have to be chopped up.

Reply:
yes, `GameControl::GetLastActor()` is the actor to use here

Reply:
ok got it